.gitlab-ci/stages/deploy.yml

```diff
@@ -44,7 +44,7 @@ include:
     CONTEXT: api
     ELASTICSEARCH_LOG_LEVEL: trace
     IMAGE_NAME: ${CI_REGISTRY_IMAGE}/api
-    PORT: 80
+    PORT: 8000
```

Comment:
why there is no ingress update too ?

Reply:
looks like the api ingress rule also use this variable